src/storage/impl/Internal.js

```diff
@@ -24,6 +27,9 @@ class Internal {
         autosave: true,
         serializationMethod: 'pretty'
       });
+      if (fs.existsSync('medmorph.db')) {
```

Comment:
The recommended approach in LokiJS to perform things immediately after initialization is to use the `autoloadCallback` option: https://github.com/techfort/LokiJS/wiki/LokiJS-persistence-and-adapters
I think the reason is the init process does some async stuff and may not be ready immediately after the constructor returns

Reply:
I tried that last night and it didn't work. To make this work I had to pass in `this` instead of requiring it. With a fresh set of eyes on it this morning I realized that's what I needed to make the callback work